app/views/service_providers/_form.html.erb

```diff
@@ -33,10 +33,16 @@
 <%= form.input :return_to_sp_url, input_html: { class: 'h4p4' }, label: "<b>Return to App URL</b><br>The applications URL which login.gov provides to users when they wish to go directly to the app's site or cancel out of authentication. For example: <code>https://app.agency.gov</code>".html_safe %>
 
 <div class="oidc-fields">
-  <%= form.input :redirect_uris, label: "<b>Redirect URIs</b> — <i>OpenID Connect only</i><br>One or more URIs that login.gov will redirect to after authentication. These can be web URLs (public, internal, or localhost) or a custom scheme to support native applications, for example: <code>gov.agency.app://result</code>".html_safe %>
-  <button id="add-redirect-uri-field" class="usa-button" type="button">Add another URI</button>
+  <%= form.label :redirect_uris, "<b>Redirect URIs</b> — <i>OpenID Connect only</i><br>One or more URIs that login.gov will redirect to after authentication. These can be web URLs (public, internal, or localhost) or a custom scheme to support native applications, for example: <code>x-example-app:/result</code>".html_safe %>
 </div>
 
+<div class="saml-fields">
+  <%= form.label :redirect_uris, "<b>Redirect URIs</b> — <i>SAML only</i><br>One or more URIs that login.gov will include in the SAML response CSP headers to allow multiple targets for redirection. These can be web URLs (public, internal, or localhost) or a custom scheme to support native applications, for example: <code>x-example-app:/result</code>".html_safe %>
```

Comment:
I'd like the label to be "_Additional_ Redirect URIs" 

Also, these hints `x-example-app:/result` should be `gov.agency.app://result`.

Reply:
Remove "OpenID Connect only" and "SAML only" from the label, too.